src/Spec-Core/TableBasePresenter.class.st

```diff
@@ -91,13 +91,32 @@ TableBasePresenter >> isShowingColumnHeaders [
 	^ showColumnHeadersHolder value
 ]
 
+{ #category : #accessing }
+TableBasePresenter >> items [
```

Comment:
this is bad.
There is a reason why `TableBasePresenter` does not understand `items`: not all its children understand them. In particular, tree understands roots and children, not items.

Reply:
So we have a problem because TableBasedPresenter calls #items.

Reply:
In #selectedIndex:

Reply:
in my version there is no selectedIndex:, which is also correct (because trees have a selectedPath, not a selectedIndexP)

Reply:
Ok, I thought you write it but now that I look at the blame I see that @guillep added this method.

Reply:
yes, and I already pushed down them